<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QWidget" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>453</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>453</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>453</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ClientWindow</string>
  </property>
  <property name="windowOpacity">
   <double>0.980000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="selectAllButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 127, 190);
font: 75 9pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectInverseButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 127, 190);
font: 75 9pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Select Inverse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 127, 190);
font: 75 9pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="emptyButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 127, 190);
font: 75 9pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Empty</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 127, 190);
font: 75 9pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>&amp;Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="bgLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 8pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Traffic monitoring system 2012</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>711</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="sendButton">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Calibri&quot;;
background-color: rgb(47, 47, 47);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>&amp;Send</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stopButton">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="fileTransferStatusLabel">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
      </property>
      <property name="text">
       <string>Transfer Status:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="minimumSize">
       <size>
        <width>460</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>fileTransferStatusLabel</zorder>
   <zorder>progressBar</zorder>
   <zorder>stopButton</zorder>
   <zorder>sendButton</zorder>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 127, 190);
font: 75 9pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="minimizeButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>0</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(97, 168, 255);
color: rgb(255, 255, 255);
font: 87 9pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>â€”</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>0</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(229, 86, 131);
color: rgb(255, 255, 255);
font: 87 9pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>751</width>
     <height>21</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 37, 113);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="fileTableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>691</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <column>
    <property name="text">
     <string>File Name</string>
    </property>
    <property name="textAlignment">
     <set>AlignHCenter|AlignVCenter|AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>File Path</string>
    </property>
    <property name="textAlignment">
     <set>AlignHCenter|AlignVCenter|AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>File Size</string>
    </property>
    <property name="textAlignment">
     <set>AlignHCenter|AlignVCenter|AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Transfer State</string>
    </property>
    <property name="textAlignment">
     <set>AlignHCenter|AlignVCenter|AlignCenter</set>
    </property>
   </column>
  </widget>
  <zorder>label</zorder>
  <zorder>selectAllButton</zorder>
  <zorder>selectInverseButton</zorder>
  <zorder>deleteButton</zorder>
  <zorder>emptyButton</zorder>
  <zorder>browseButton</zorder>
  <zorder>bgLabel_2</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>closeButton</zorder>
  <zorder>minimizeButton</zorder>
  <zorder>ExitButton</zorder>
  <zorder>fileTableWidget</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>selectAllButton</tabstop>
  <tabstop>selectInverseButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ExitButton</sender>
   <signal>clicked()</signal>
   <receiver>ClientWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>729</x>
     <y>4</y>
    </hint>
    <hint type="destinationlabel">
     <x>733</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
